libs/application/ui-fields/src/lib/CheckboxFormField/CheckboxFormField.tsx

```diff
@@ -16,8 +16,12 @@ import { useLocale } from '@island.is/localization'
 
 import { getDefaultValue } from '../../getDefaultValue'
 
+interface CheckboxFieldPropsExtra extends CheckboxField {
+  rightContent?: React.ReactNode
+}
+
 interface Props extends FieldBaseProps {
-  field: CheckboxField
+  field: CheckboxFieldPropsExtra
```

Comment:
This shouldnt be the way to solve this, you should add this prop to the CheckBoxField in @island.is/application/types.